reconcile/utils/rosa/session.py

```diff
@@ -124,7 +124,26 @@ def create_hcp_cluster(
         data is highlighted as errors.
         """
         return self.cli_execute(
-            cmd=rosa_hcp_creation_script(cluster_name, spec, dry_run),
+            cmd=generate_rosa_creation_script(cluster_name, spec, dry_run),
+            check_interval_seconds=check_interval_seconds,
+            timeout_seconds=timeout_seconds,
+        )
+
+    def create_rosa_cluster(
```

Comment:
i think there is no difference left between this function and `create_hcp_cluster`. how about we delete `create_hcp_cluster`?

Reply:
Absolutely!